src/server/classroom/internal.controller.js

```diff
@@ -26,7 +28,7 @@ async function sendMessage(packet, socket, io) {
 
   classroom.messages.push(message);
   await classroom.save();
-  socket.emit('new incoming message', message.filterSafe());
+  socket.emit('new message', message.filterSafe());
```

Comment:
Make sure the server respond
```suggestion
  socket.emit('new message', message.filterSafe());
  return callback({});
```